docs/public/static/error-codes.json

```diff
@@ -19,5 +19,6 @@
   "18": "MUI: The provided shorthand %s is invalid. The format should be `@<breakpoint | number>` or `@<breakpoint | number>/<container>`.\nFor example, `@sm` or `@600` or `@40rem/sidebar`.",
   "19": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150.",
   "20": "MUI: `vars` is a private field used for CSS variables support.\nPlease use another name or follow the [docs](https://mui.com/material-ui/customization/css-theme-variables/usage/) to enable the feature.",
-  "21": "MUI: The `colorSchemes.%s` option is either missing or invalid."
+  "21": "MUI: The `colorSchemes.%s` option is either missing or invalid.",
+  "22": "MUI: `vars` is a private field used for CSS variables support.\nPlease use another name or follow the [docs](https://v6.mui.com/material-ui/customization/css-theme-variables/usage/) to enable the feature."
```

Comment:
Do we need this new 22 error code or can we just use the 20 one pointing to the latest (v7) docs?